extension-localization/src/main/java/com/mapbox/maps/extension/localization/Localization.kt

```diff
@@ -25,6 +33,21 @@ internal fun setMapLanguage(locale: Locale, style: StyleInterface, layerIds: Lis
     return
   }
 
+  /**
+   * Check if the server side localization is set and Log the error if present.
+   * `localizeLabel` method (ie Style localization) will not work with server side localization
+   */
+  checkServerSideLocalizationSet { isLocalizationSet ->
```

Comment:
this is async call. Even if `isLocalizationSet` is true, we are trying to do client-side localization.

Reply:
for now yes. but we can discuss if we want to `block the client-side localization, if server-side localization is present`, wdyt @mapbox/maps-android.